fabric-tool-attribute-api-v1/src/main/java/net/fabricmc/fabric/api/tool/attribute/v1/ItemStackContext.java

```diff
@@ -0,0 +1,9 @@
+package net.fabricmc.fabric.api.tool.attribute.v1;
+
+import org.jetbrains.annotations.Nullable;
+
+import net.minecraft.entity.LivingEntity;
+
+public interface ItemStackContext {
+	void fabricToolAttributes_setContext(@Nullable LivingEntity contextEntity);
```

Comment:
was this meant to be in api package?

Reply:
No, I just don't know where to put it lol